setup/git/000_EnvSetup.rb

```diff
@@ -103,10 +103,12 @@
     arch = host[:ruby_arch] || 'x86'
     step "#{host} Selected architecture #{arch}"
 
+    # TODO we need to be able to branch here. The PA 1.10.x stream needs to use
+    # 2.1.x, and the rest uses 2.4.x
     revision = if arch == 'x64'
-                 '2.1.x-x64'
+                 '2.4.x-x64'
                else
-                 '2.1.x-x86'
+                 '2.4.x-x86'
```

Comment:
I pulled this change from https://github.com/puppetlabs/puppet/commit/b548568378841a4497c3238f2f665bc8b4b9617a#diff-77dcb7e76572173a2cfb93ebbf6a5ceb